content/en/blog/2022/apisix.md

```diff
@@ -0,0 +1,389 @@
+-H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' \
+-X PUT -d '{
+    "plugins": {
+        "opentelemetry": {
+            "sampler": {
+                "name": "always_on"
+            }
+        }
+    }
+}'
+```
+
+### Method 3: Customize Labels for Span through additional_attributes
+
+For the configuration of `sampler` and `additional_attributes`, you can refer to
+the [Apache APISIX official documentation](https://apisix.apache.org/docs/apisix/next/plugins/opentelemetry/#attributes),
+where `additional_attributes` is a series of `Key:Value` pairs, you can use it
+to customize the label for Span, and can follow Span to display on the Web UI.
+Add `route_id` and `http_x-custom-ot-key` to the span of a route through
+`additional_attributes`, you can refer to the following configuration:
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1001 \
+  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' \
+  -X PUT -d '
+{
+    "uri": "/put",
+    "plugins": {
+        "opentelemetry": {
+            "sampler": {
+                "name": "always_on"
+            },
+            "additional_attributes":[
+                "route_id",
+                "http_x-custom-ot-key"
+            ]
+        }
+    },
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "httpbin.org:80": 1
+        }
+    }
+}'
+```
+
+## Test and Verify the Plugin
+
+You can enable `opentelemetry` plugin in any of the above three methods. The
+following example uses the example of method three to create a route. After the
+creation is successful, you can refer to the following commands to access the
+route:
+
+```shell
+curl -X PUT -H `x-custom-ot-key: test-ot-val` http://127.0.0.1:9080/put
+```
+
+After the access is successful, you can see the details of the span similar to
+`/put` in the Jaeger UI, and you can see that the custom tags in the route are
+displayed in the Tags list: `http_x-custom-ot-key` and `route_id`.
+
+![Jaeger UI](/img/apisix/jaeger-ui-1.png)
+
+You need to note that the `additional_attributes` configuration is set to take
+values from Apache APISIX and Nginx variables as `attribute` values, so
+`additional_attributes` must be a valid Apache APISIX or Nginx variable. It also
+includes HTTP Header, but when fetching http_header, you need to add `http_` as
+the prefix of the variable name. If the variable does not exist, the `tag` will
+not be displayed.
+
+## Example
+
+This scenario example deploys Collector, Jaeger, and Zipkin as backend services
+by simply modifying the official example of OpenTelemetry Collector, and starts
+two sample applications (Client and Server), where Server provides an HTTP
+service, and Client will cyclically call the server provided by the server. HTTP
+interface, resulting in a call chain consisting of two spans.
+
+### Step 1: Deploy OpenTelemetry
+
+The following uses docker-compose as an example. For other deployments, please
+refer to the OpenTelemetry official documentation.
```

Comment:
Change the last sentence to something like:

> For other deployments, see ADD-A-LINK-HERE

Reply:
Updated.